### DISCUSSION TITLE
ft-fms-docker for FMS 19.4.2

### DISCUSSION THREAD

**Discussion by kietnt072** [fmgarage/ft-fms-docker#14] [Ideas]
Hi fmgarage,

I tried to change something related to the filemaker_server package in .env, install.sh, helper_ubuntu.sh and copy new .deb file for FMS 19.4.2.
It worked, but I did not know what were going wrong with the container such as port 16000 for admin-console did not run as usual instead of 16000, admin-console run on port 443,..it has about 20 medium vulnerabilities after I scan Docker.

Please arrange your time to release new update for FMS 19.4.2

Thank you very much

**Comment by sk-gara** [comment]
Hi,

Claris changed the port for the admin console to the https port 443, so that is correct behaviour.

What did you need to change to run fms 19.4.2? And could you post the results of your scan? Did you you do a scan prior to 19.4.2 with no or less reported vulnerabilities?
Also, which OS do you use?

Greetings,
Steffen

**Reply by kietnt072** [reply]
Thanks, Steffen

First of all, pls change the document that your solution office support FMS 19.4.2. About the vulnerabilities, I have some pics (scanned 3 days ago). 

Have a good day.
Thanks

<img width="1408" alt="Screen Shot 2022-02-15 at 13 19 09" src="https://user-images.githubusercontent.com/42569231/154005418-443b6c75-1a0e-4e24-961a-edcd60fccc10.png">
<img width="1388" alt="Screen Shot 2022-02-15 at 13 19 24" src="https://user-images.githubusercontent.com/42569231/154005441-23d6c686-b73d-4a34-9a8e-fdebd1e927e0.png">

**Comment by sk-gara** [comment]
Hello,

I changed the part in the README, thanks for pointing out.

As for the scan results, the vulnerabilities found are in Ubuntu system packages and node, amongst others, and I don't maintain these. I'm sure they were there in prior images, since they are independent from the fms version installed. You could scan the official ubuntu:18.04 image to double check.

So long,
Steffen

**Reply by kietnt072** [reply]
Thank you vey much Steffen,

After scanning Official ubuntu:18.04 , it has 4 medium vulnerabilities

Thanks
<img width="1320" alt="Screen Shot 2022-02-17 at 15 10 43" src="https://user-images.githubusercontent.com/42569231/154433162-508dc90c-1a0d-44c4-aada-8d0da16e45eb.png">

**Reply by sk-gara** [reply]
The additional vulnerabilities come with the dependencies of fms, so there's no way around I'm afraid.

**Comment by ariley** [comment]
Hi,

I tried to follow Claris' and your instructions and I keep on having unmet dependencies.[
(url
<img width="1204" alt="Screen Shot 2022-08-07 at 11 08 13 AM" src="https://user-images.githubusercontent.com/381123/183304940-8c0778db-dea5-43c1-aafc-d30f81383bfd.png">
)

Any ideas?

Thanks.

**Reply by nilswald** [reply]
Hi Agnes,
is this on Apple Silicon? 
Nils

**Reply by ariley** [reply]
Yes, it is: M1.

**Reply by nilswald** [reply]
Unfortunately this will not work because you would need an ARM version of FMS for Linux, but I am pretty sure this is already on the roadmap. We are desperately waiting for this so we can use it with Multipass, which is a much cooler way to run Ubuntu on a Mac. (see https://github.com/fmgarage/ft-fms-multipass)

**Reply by ariley** [reply]
Thank you for your reply.
